arq/cli.py

```diff
@@ -60,7 +60,7 @@ async def watch_reload(path: str, worker_settings: 'WorkerSettingsType') -> None
     except ImportError as e:  # pragma: no cover
         raise ImportError('watchfiles not installed, use `pip install watchfiles`') from e
 
-    loop = asyncio.get_event_loop()
+    loop = asyncio.get_running_loop()
```

Comment:
Pipelines fixed themselves, so I don't think we need to merge this PR. 

This is somewhat still relevant, but also not really needed. Feel free to close. 

> Another potential good thing to do is to use get_running_loop() over get_event_loop(), as per [documentation](https://docs.python.org/release/3.10.9/library/asyncio-eventloop.html#asyncio.get_event_loop).